packages/clay-color-picker/src/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -1233,7 +1233,7 @@ exports[`Rendering renders w/ var() 1`] = `
             aria-label="Color selection is var(--blue)"
             class="form-control input-group-inset input-group-inset-before"
             type="text"
-            value="VAR(--BLUE)"
+            value="var(--blue)"
```

Comment:
@sergiojimcos do you know why this needs to be updated? not saying it's wrong but I'm curious about the change.

Reply:
Sure @julien. At first I changed this snapshot when I modified the values of the input into upper case. As I told you yesterday, finally the value of the input must be in lower case, for that reason I changed into into the default one